pkgs/development/python-modules/mahotas/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage, fetchFromGitHub, nose, pillow, scipy, numpy, imread, stdenv }:
+
+buildPythonPackage rec {
+  pname = "mahotas";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "luispedro";
+    repo = "mahotas";
+    rev = "v${version}";
+    sha256 = "1d2hciag5sxw00qj7qz7lbna477ifzmpgl0cv3xqzjkhkn5m4d7r";
+  };
+
+  patches = [ ./disable-impure-tests.patch ];
```

Comment:
How do you know that impurity is causing those test failures? Did you report that upstream?

Reply:
I ran the build multiple times in a sandboxed environment and got diffferent results in the check phase (sometimes both tests broke, sometimes one of them succeeded).

Not yet, but what does @luispedro (package maintainer and author) think about this?:)

Reply:
It seems like he's not very active on github recently, so maybe its better to open an issue on the project instead. Then you can link the issue in a comment and explain that the failures are transient and we can get this merged.

Reply:
done! I submitted an issue report and documented it in the package expression:)

Reply:
Thanks a lot :)

Reply:
Thanks guys for all this work. I have just moved jobs, so I don't have as easy of a setup to test this ATM, but since you opened an issue on the main mahotas tracker it won't get lost.